### PR TITLE
Add support for more chains

### PR DESCRIPTION
**Pull request by ntindle**
I absolutely missed something here when setting this up

 Mention [stepsize] in a comment if you'd like to report some technical debt. See examples here.

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -141,7 +161,14 @@ func (c *Client) GetIPFSApiAddress() string {
...
 	_, err := os.Stat(env_path)
 	if errors.Is(err, os.ErrNotExist) {
-		return IPFS_API_ADDRESS
+		switch c.clientMode {
+		case mt.ClientMode_ENDPOINT_BETA:
+			return IPFS_ADDRESS_BETA
+		case mt.ClientMode_ENDPOINT_DEV:
+			return IPFS_ADDRESS_DEV
+		case mt.ClientMode_ENDPOINT_LOCAL:
+			return IPFS_ADDRESS_LOCAL
```
Comment (prdlk) [inline pkg/client/client.go:179]: Add prod as an option for client mode here

File: pkg/client/client.go [hunk 2]
```diff
@@ -124,7 +137,14 @@ func (c *Client) GetIPFSAddress() string {
 	// by default use .env if it exists
 	_, err := os.Stat(env_path)
 	if errors.Is(err, os.ErrNotExist) {
-		return IPFS_ADDRESS
+		switch c.clientMode {
+		case mt.ClientMode_ENDPOINT_BETA:
+			return IPFS_ADDRESS_BETA
+		case mt.ClientMode_ENDPOINT_DEV:
+			return IPFS_ADDRESS_DEV
+		case mt.ClientMode_ENDPOINT_LOCAL:
```
Comment (prdlk) [inline pkg/client/client.go:152]: Add prod as an option for client mode here

**Review by prdlk — CHANGES_REQUESTED** [review]
The InitializeRequest from the Motor client supplies the mode that the node should use - pkg/motor uses this switch statement - so after adding that as a case this will work. Also build the protos using `make proto.go` so that the updated request contains the new prod option.

File: pkg/client/client.go (hunk 1 above)
Comment (ntindle) [reply pkg/client/client.go:179]: done

File: pkg/client/client.go (hunk 2 above)
Comment (ntindle) [reply pkg/client/client.go:152]: done

**Review by joshLong145 — APPROVED** [review]

**Review by prdlk — APPROVED** [review]

**Pull request merged**